Lombiq.VueJs.csproj

```diff
@@ -39,6 +39,12 @@
     <Folder Include="wwwroot\" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="3.3.0">
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
+  </ItemGroup>

```

Comment:
This is not necessary. If you add the Analyzers project [as documented](https://github.com/Lombiq/.NET-Analyzers#how-to-add-the-analyzers-to-you-project-repository) then this package and all the other analyzers will automatically be included in all projects of the given solution. If you use our [OSOCE](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions) repo for developing these modules that happens as well.

Reply:
Tha's strange huh. This was added automatically by your Analyzers project I think. Pretty sure I followed the documentation. I will remove - thanks for the advice.